app/models/user.rb

```diff
@@ -575,6 +623,74 @@ def flipper_id
 
   private
 
+  # rubocop:disable Metrics/BlockLength
+  def migrate_users_fields_to_users_settings(users_setting_record)
```

Comment:
@msarit this method does not look like it uses the newly updated user record values (obtained from self in the after_commithook) to update the user settings table?